spec/models/social_networking/comment_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "spec_helper"
+
+module SocialNetworking
+  describe Comment, type: :model do
+    fixtures(:participants, :"social_networking/goals")
+    let(:participant1) { participants(:participant1) }
+    let(:goal) { social_networking_goals(:participant2_goal_alpha) }
+
+    it "should return the number of comments made today" do
+      count = Comment.for_today.count
+
+      Comment.create(
+        participant_id: participant1.id,
+        item_id: goal.id,
```

Comment:
Also, if you use `item: goal` here, you won't have to set the `item_type` attribute.